solr/solr-ref-guide/modules/configuration-guide/pages/index-segments-merging.adoc

```diff
@@ -250,7 +250,7 @@ This is not required for near real-time search, but will reduce search latency o
 == Compound File Segments
 
 Each Lucene segment is typically comprised of a dozen or so files.
-Lucene can be configured to bundle all of the files for a segment into a single compound file using a file extension of `.cfs`, for "Compound File Segment".
+Solr can be configured to bundle all of the files for a Lucene segment into a single compound file using a file extension of `.cfs`, for "Compound File Segment".
```

Comment:
This may be controversial, since this is a pure lucene-level setting. However, from the user's perspective you configure this in **Solr**, so I feel this sounds better. See the paragraph in context here https://nightlies.apache.org/solr/draft-guides/solr-reference-guide-nightly/solr/9_0/configuration-guide/index-segments-merging.html#compound-file-segments